bigtop_toolchain/manifests/installer.pp

```diff
@@ -14,6 +14,7 @@
 # limitations under the License.
 
 class bigtop_toolchain::installer {
+  include bigtop_toolchain::jdk11
   include bigtop_toolchain::jdk
```

Comment:
Hmm. /usr/bin/java points to Java 11 after installing both OpenJDK 8 and OpenJDK 11 on Ubuntu. If we want to retain existing setup in which OpenJDK 8 is preferred as default, the alternatives should be switched back.

```
$ update-alternatives --display java
java - auto mode
  link best version is /usr/lib/jvm/java-11-openjdk-amd64/bin/java
  link currently points to /usr/lib/jvm/java-11-openjdk-amd64/bin/java
  link java is /usr/bin/java
  slave java.1.gz is /usr/share/man/man1/java.1.gz
/usr/lib/jvm/java-11-openjdk-amd64/bin/java - priority 1111
  slave java.1.gz: /usr/lib/jvm/java-11-openjdk-amd64/man/man1/java.1.gz
/usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java - priority 1081
  slave java.1.gz: /usr/lib/jvm/java-8-openjdk-amd64/jre/man/man1/java.1.gz
```

Reply:
Thank you, I have to check and fix this. Using `update-java-alternatives` or `update-alternatives --config java` seems fine.